README.md

```diff
@@ -64,21 +63,21 @@ Using GitLink via the command line is very simple:
 
 When working with a content proxy or an alternative git VCS system that supports direct HTTP access to specific file revisions use the `-u` parameter with the custom raw content root URL
 
-    GitLink.exe c:\source\catel -u https://raw.githubusercontent.com/catel/catel
+    GitLink.exe -u https://raw.githubusercontent.com/catel/catel <pdbfile>
```

Comment:
I very much agree with that. That line is first. I've reordered the parameters for the longer command lines. In fact, I wouldn't mind dropping the log to file doc since it's in the usage doc and may not be that interesting to make a README longer. What do you think>